e2e/__tests__/__snapshots__/detectOpenHandles.ts.snap

```diff
@@ -1,5 +1,26 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`deals with http servers and promises 1`] = `
+Jest has detected the following 1 open handle potentially preventing Jest from exiting:
+Of them 1 was collected within 100ms of the tests completing.
```

Comment:
this probably isn't clear enough...

Reply:
How about:
```
Detected 3 open handles potentially preventing Jest from exiting:

1. <red>DNSCHANNEL (uncollected, potential leak)
2. <red>Timeout (uncollected, potential leak)
3. <yellow>TCPSERVERWRAP (collected 100ms after test run, potentially spawned other uncollected handles)</yellow>

      4 |   const server = http.createServer();
      5 |   await new Promise(resolve => {
    > 6 |     server.listen(resolve);

[optionally] There were no uncollected handles - this is unexpected if your tests do not exit cleanly.
```
Not sure about the "optional" part. Shouldn't it say something like "Collected handles shouldn't prevent Jest from closing. Try something else"?

Reply:
I like it!

Reply:
![image](https://user-images.githubusercontent.com/1404810/74926744-a97efd00-53d6-11ea-9dd4-b0a80899b741.png)

thoughts? I kept the heading, since if there were many collected, it looked really noisy...

Reply:
I pushed that, since it's an improvement regardless

Reply:
Like it

Reply:
What do you think about moving the "Of them 1 was collected..." below, so it separates collected from uncollected more obvoiusly?
```
Detected 6 open handles potentially preventing Jest from exiting:

...

1 of them was collected within 100ms after the test run completed:

...
```
I'm also not a fan of:
> Jest has detected the following 1 open handle potentially preventing Jest from exiting

as it contains "Jest" twice. 
This is shorter and gets to the point:
> Detected 6 open handles potentially preventing Jest from exiting